gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineErrorProne.java

```diff
@@ -216,6 +216,11 @@ private static void configureErrorProneOptions(
             // Don't apply refaster to itself...
             return;
         }
+        // Hint to error-prone that we're compiling test code
+        if (javaCompile.getName().endsWith("compileTestJava")) {
```

Comment:
Any idea how we can do this better?

Reply:
It seems like this is already being done for us: https://github.com/tbroyer/gradle-errorprone-plugin/blob/master/src/main/kotlin/net/ltgt/gradle/errorprone/ErrorPronePlugin.kt#L112

Reply:
mmm you'd think that but I don't think we've updated that plugin yet https://github.com/palantir/gradle-baseline/blob/2a560566527c34f707f5b78bc54345d213a2b08d/versions.props#L34

Reply:
Cool, lets upgrade then! I think we can decouple that change from updating the errorprone rules to respect the flag

Reply:
👍